docs/manuals/gui/viselements/charts/gantt.md

```diff
@@ -1,12 +1,139 @@
-## Gantt charts
+# Gantt charts
 
-Although real Gantt charts are not supported by Plotly, there are ways to display horizontal bar charts with different starting points that can look like a Gantt chart.
+Although Plotly does not have Gantt chart-specific features, there are ways to display horizontal bar charts with different starting points that can look like a Gantt chart.
 
-### Simple Gantt like chart
+Therefore, the basis for creating a chart control representing a Gantt chart, you must
+set the [*type*](../chart.md#p-type) property of the control to "bar" and set the
+[*orientation*](../chart.md#p-orientation) property to "h".
 
-Using a horizontal bar chart with different starting points will look like a Gantt chart.
-- _type_ property is set to "bar".
-- _orientation_ property is set to "h" or "horizontal".
+In a Gantt chart, you have two inescapable traits:
+
+- the *x* axis represents the time.
+- the *y* axis holds a list of tasks that are scheduled or a list
+    of resources that are assigned different tasks along the time.
+
+Gantt charts are based on a timeline. Working with the Plotly bar charts using
+dates and times as an axis value is pretty tricky and needs explanations:
+
+- Every bar represents a time span for a given activity.
+- Setting its *y* location is straightforward: the *y* value for that bar is the name of
+    the task or the resource it is assigned to.
+- Each bar needs two values, indicating when an activity starts and ends. This
+    is where things are a bit more complicated:
+    - The *base* value for each bar must be set to the date when the activity ends.
+    - The *x* value for each bar must be set to a `datetime` value representing
+        the negative duration of the activity relative to January 1st, 1970.
+
+To summarize: if you have a task "T" planned between &lt;date1&gt; and &lt;date2&gt;,
+you will have to provide the chart control with:
+
+- *base* = &lt;date2&gt;
```

Comment:
I thought you could set base as date1 and x as date2 -date1 (+ 1/1/70)

Reply:
I tried that... does not work.
Well... it produces new dates, that are not the ones I want.